extension/src/shared/reearth/hooks/useLayer.ts

```diff
@@ -30,6 +32,7 @@ export const useLayer = ({
       type: "simple",
       data: data,
       events,
+      defines: defines,
```

Comment:
You need to pass this in [here](https://github.com/eukarya-inc/PLATEAU-VIEW-3.0/pull/172/files#diff-d7246fd453668bec2c79394f62632e08671fbbb796536d152f8f7e71b835b797R54) as well.

Reply:
Done in https://github.com/eukarya-inc/PLATEAU-VIEW-3.0/pull/172/commits/9808906e8f5f1f67760594be85a10eecf4b020bf